sunpy/physics/tests/test_differential_rotation.py

```diff
@@ -130,6 +130,11 @@ def test_snodgrass(seconds_per_day):
     assert_quantity_allclose(rot, 135.4232 * u.deg, rtol=1e-3)
 
 
+def test_rigid(seconds_per_day):
+    rot = diff_rot(10 * seconds_per_day, 30 * u.deg, rot_type='rigid')
```

Comment:
```suggestion
    rot = diff_rot(10 * seconds_per_day, [0, 30, 60] * u.deg, rot_type='rigid')
```
Might as well test multiple latitudes to confirm that the output doesn't vary with latitude